en_US/messaging/mqtt-shared-subscription.md

```diff
@@ -31,11 +31,9 @@ Shared subscription topics prefixed with `$queue/` are for subscribers not in gr
 
 ## Shared Subscription and Session
 
-The concept of shared subscription and the use of a persistent session in MQTT clients presents a contradiction, making it impossible to use both features simultaneously. If you are using the shared subscription functionality, it is essential to enable the clean session feature on the client by setting the `clean_session` parameter to `true`.
+When a client has a persistent session and subscribes to shared subscriptions, the session continues to receive messages published to the shared subscription topics while the client disconnects. If the client stays disconnected for a long time and the message publishing rate is high, the internal message queue in the session state may overflow. To avoid this problem, it is recommended to use a clean session ( `clean_session=true` ) for shared subscriptions. A clean session expires immediately after the client disconnects.
```

Comment:
do we have a solution for a client who is both publisher and subscriber? 

I understand in most cases the client is subscriber only.

Reply:
I think the correct scenario is the client should come back in time? We assume "If the client stays disconnected for a long time ..." and I think it is not EMQX's problem as the broker.

Reply:
> do we have a solution for a client who is both publisher and subscriber?
> 
> I understand in most cases the client is subscriber only.

It’s hard to imagine any practical use for one client to be both publisher and shared-subscriber of the same topic